contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -700,6 +701,11 @@ library ValidationLogic {
             Errors.INVALID_ASSET_TYPE
         );
 
+        require(
+            params.liquidator != params.borrower,
```

Comment:
why do we need to pass `liquidator` instead of just using msg.sender?

Reply:
is there any difference?